pandas/core/arrays/categorical.py

```diff
@@ -379,6 +382,110 @@ def __init__(
         self._dtype = self._dtype.update_dtype(dtype)
         self._codes = coerce_indexer_dtype(codes, dtype.categories)
 
+    @classmethod
+    def from_dummies(cls, dummies: "DataFrame", ordered=None):
+        """
+        Create a `Categorical` using a ``DataFrame`` encoding those categories
+        as dummy/ one-hot encoded variables.
```

Comment:
The [docstring guide](https://pandas.pydata.org/docs/development/contributing_docstring.html) asks that the short summary fit on a single line, is there a way to reword it?